### PR TITLE
Add release notes for 3.11 RC1

### PR DESCRIPTION
**Pull request by Griffin-Sullivan**
We'll need to go through these and make sure they all got into the RC because I suspect some of them weren't included.

### CODE REVIEW

File: _includes/manuals/3.11/1.2_release_notes.md [hunk 1]
```diff
@@ -4,19 +4,206 @@ This section will be updated prior to the next release.
...
+#### Greatly decreased JavaScript size for plugins
+
+Previously, Foreman's whole JavaScript bundle was duplicated in every plugin.
+Now a separate bundle is generated that Foreman and each plugin can reuse.
+In most plugins, we saw a 2 to 3 MB reduction in size and depending on the number of plugins, this can save a significant amount of space.
+While Foreman does compress and cache these JavaScript bundles, they still had to be loaded all the time.
+
+For more details, see https://projects.theforeman.org/issues/37252 and https://projects.theforeman.org/issues/37391.
+
+#### Running Foreman on {EL} 9 is fully supported
```
Comment (evgeni) [inline _includes/manuals/3.11/1.2_release_notes.md]: EL is not a macro that gets properly expanded here

File: _includes/manuals/3.11/1.2_release_notes.md [hunk 2]
```diff
@@ -4,19 +4,206 @@ This section will be updated prior to the next release.
...
+
+Foreman 3.10 only supported {EL} 9 as experimental, but with this release it is fully supported.
+
 ### Upgrade warnings
 
-#### PostgreSQL upgraded to 13 on Enterprise Linux 8
+#### keycloak-httpd-client-install dropped from {EL} 9
```
Comment (evgeni) [inline _includes/manuals/3.11/1.2_release_notes.md]: Same EL

File: _includes/manuals/3.11/1.2_release_notes.md [hunk 3]
```diff
@@ -4,19 +4,206 @@ This section will be updated prior to the next release.
...
+The version in {EL} 9 contains all the required features but is older than what Foreman has shipped.
+Foreman 3.10 was the first release on {EL} 9 and it was marked as experimental.
+As a result, the decision has been made to remove it from Foreman's {EL} 9 packaging.
+Users who have this package installed should downgrade it using `dnf downgrade keycloak-httpd-client-install`.
+
+For more details, see https://projects.theforeman.org/issues/37334.
 
 ### Deprecations
 
+#### Running Foreman on {EL} 8 removal in Foreman 3.13
```
Comment (evgeni) [inline _includes/manuals/3.11/1.2_release_notes.md]: EL! ;)

File: _includes/manuals/3.11/1.2_release_notes.md (hunk 1 above)
Comment (Griffin-Sullivan) [reply _includes/manuals/3.11/1.2_release_notes.md]: Copy paste from `foreman-documentation`. What can I used instead?

Comment (evgeni) [reply _includes/manuals/3.11/1.2_release_notes.md]: I'd use "Enterprise Linux", in `foreman-documentation` we expand it to either "Enterprise Linux" or "Red Hat Enterprise Linux", depending on the type of build (upstream vs satellite), but here we don't have to do the same differentiation

Comment (Griffin-Sullivan) [reply _includes/manuals/3.11/1.2_release_notes.md]: Ok addressed that

**Review by ekohl — APPROVED** [review]

**Pull request merged**